### PR TITLE
add torchmetrics

### PR DESCRIPTION
**Pull request by Borda**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/torchmetrics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - torchmetrics
+
+about:
+  summary: PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers. Scale your models. Write less boilerplate.
+
+about:
+  home: https://pypi.org/project/torchmetrics/
```
Comment (hadim) [inline recipes/torchmetrics/meta.yaml:56]: the `about` section is duplicated.

File: recipes/torchmetrics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Use the SPDX identifier, e.g: GPL-2.0-only instead of GNU General Public License version 2.0
+  # See https://spdx.org/licenses/
+  license: Apache-2.0
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
```
Comment (hadim) [inline recipes/torchmetrics/meta.yaml:63]: You can remove all the comments to make the recipe more readable.

File: recipes/torchmetrics/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
```
Comment (hadim) [inline recipes/torchmetrics/meta.yaml:30]: You can mark the package as `noarch: true`.

Comment (Borda) [inline recipes/torchmetrics/meta.yaml:30]:
```suggestion
  noarch: true
```

File: recipes/torchmetrics/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    - pytorch >=1.3
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - torchmetrics
+
+about:
+  summary: PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers. Scale your models. Write less boilerplate.
```
Comment (Borda) [inline recipes/torchmetrics/meta.yaml:53]:
```suggestion
```

**Comment by Borda** [comment]
cool, thank you @hadim

**Comment by Borda** [comment]
@conda-forge/staged-recipes seems all is set now 🐰

File: recipes/torchmetrics/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/torchmetrics/meta.yaml:1]: Please remove the comments.

File: recipes/torchmetrics/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
```
Comment (Borda) [inline recipes/torchmetrics/meta.yaml:2]:
```suggestion
```

File: recipes/torchmetrics/meta.yaml (hunk 5 above)
Comment (Borda) [reply recipes/torchmetrics/meta.yaml]: done :]

File: recipes/torchmetrics/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    - pip
+    - python >=3.6
+  run:
+    - packaging
+    - python >=3.6
+    - pytorch >=1.3
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
```
Comment (Borda) [inline recipes/torchmetrics/meta.yaml:31]:
```suggestion
```

File: recipes/torchmetrics/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,55 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (Borda) [inline recipes/torchmetrics/meta.yaml:3]:
```suggestion
```

File: recipes/torchmetrics/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 481a28759acd2d77cc088acba6bc7dc4a356c7cb767da2e1495e91e612e2d548
+
+build:
+  noarch: true
```
Comment (chrisburr) [inline recipes/torchmetrics/meta.yaml:13]:
```suggestion
  noarch: python
```

File: recipes/torchmetrics/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+    implemented without having to install Pytorch Lightning (even though we would love for you to try it out).
+    We currently have around 25+ metrics implemented and we continuously is adding more metrics, both within
+    already covered domains (classification, regression ect.) but also new domains (object detection ect.).
+    We make sure that all our metrics are rigorously tested such that you can trust them.
+  doc_url: https://torchmetrics.readthedocs.io/
+  dev_url: https://github.com/PyTorchLightning/metrics
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (chrisburr) [inline recipes/torchmetrics/meta.yaml:48]:
```suggestion
```

**Comment by Borda** [comment]
@ocefpaf @chrisburr that you for your suggestions, all shall be set now :]

**Pull request merged**